lib/immutable-struct.rb

```diff
@@ -81,8 +81,25 @@ def self.new(*attributes,&block)
         end
       end
 
+      @@aliases = {}
+
+      def self.alias_attribute(aliaz, attr, &block)
+        @@aliases.store(aliaz, [attr, block])
+      end
+
+      def method_missing(m, *_args, &_block)
```

Comment:
Should I override `respond_to?` also?

Reply:
Actually consider `def respond_to_missing?`

Reply:
Thanks for the correction 👍